recipes/blueskyapi/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "blueskyapi" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2235abaa5c979566b98697a8000194452962ed58a82e96fbe68a83b4124e93d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7.1
+    - pip
+    - poetry-core
+  run:
+    - pandas >=1.1,<2.0
+    - python >=3.7.1
+    - requests >=2.0,<3.0
+
+test:
+  imports:
+    - blueskyapi
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: "https://blueskyapi.io"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Client for blueskyapi.io"
+  doc_url: "https://{{ name }}.readthedocs.io/en/{{ version }}/"
```

```suggestion
  doc_url: https://blueskyapi.io/docs/api
```

That seems to be the new URL.

---

That's a more general documentation for our API. The one for this package specifically should be correct as is.

---

Ahh ok. The docs URL is pretty hardcoded in some places so it would be better to have a non-version specific one. Additionally, jinja variables outside of the source sometimes trip up the bot so please hard code it.

Something like this: https://blueskyapi.readthedocs.io/en/latest/

---

Okay sure, I updated it to a fixed url :)